crane/template.go

```diff
@@ -0,0 +1,105 @@
+package crane
+
+import (
+	"fmt"
+	"github.com/michaelsauter/crane/print"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"text/template"
+)
+
+type TemplateData struct {
+	Graph              DependencyGraph
+	TargetedContainers Containers
+}
+type TemplateDataForContainer struct {
+	Graph              DependencyGraph
+	TargetedContainers Containers
+	CurrentContainer   Container
```

Comment:
I guess we should reduce the amount of stuff exposed in `Container` - side-effect functions like `Run()` or `Kill()` should not be available.